checker/checker.js

```diff
@@ -25,14 +24,26 @@ gets fired from github pr creation/update webhook.
 const valid_pr_events = ['opened', 'reopened', 'synchronize'];
 
 async function main (params) {
-  if (!params.pull_request || !valid_pr_events.includes(params.action)) {
+  const isMergeQueue = params.merge_group && params.action === 'checks_requested';
+  if (!isMergeQueue && (!params.pull_request || !valid_pr_events.includes(params.action))) {
     return {
       statusCode: 202,
       body: 'Not a pull request being (re)opened or synchronized, ignoring'
     };
   }
   let github;
   const ow = openwhisk();
+
+  if (isMergeQueue) {
+    const res = await set_green_is_bot(ow, {
```

Comment:
We actually don't wanna call `set_green_is_bot` because this handles when a PR is opened by the bot, and will end up giving back this result, like what happened [here](https://github.com/adobe/cla-bot-playground/runs/30408370895):
> Pull request issued by a bot account, carry on.

Instead, we should add a dedicated, but similar function, like so:
```ts
async function set_green_has_signed_cla (ow, args) {
  let result;
  try {
    result = await ow.actions.invoke({
      name: utils.SETGITHUBCHECK,
      blocking: true,
      result: true,
      params: {
        installation_id: args.installation_id,
        org: args.org,
        repo: args.repo,
        sha: args.commit_sha,
        status: 'completed',
        start_time: args.start_time,
        conclusion: 'success',
        title: 'CLA Signed',
        summary: 'A Signed CLA has been found for the GitHub.com user ' + args.user
      }
    });
  } catch (e) {
    return utils.action_error(e, 'Error invoking setgithubcheck when CLA was found.');
  }
  return {
    statusCode: 200,
    body: result.title
  };
}
```